crypto/dh/README

```diff
@@ -0,0 +1,21 @@
+How to use dh-pg-ka.c
+
+1) a. Compile wolfSSL with  ./configure, run
```

Comment:
This also requires `--enable-keygen` / `WOLFSSL_KEY_GEN`.

Reply:
Fixed. Added FFDHE flags as well.